app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -41,19 +42,43 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Title"
               name="title"
+              placeholder="Enter title"
             />
             <Textarea
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              placeholder="Enter note description"
+              rows={2}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="contact"
+              options={CONTACT}
+              placeholder="Select Role"
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tag"
```

Comment:
So in your business logic why exactly aren't we allowed to choose multiple tags and associate it with a note?